src/test/rustdoc-ui/cfg-test.rs

```diff
@@ -2,18 +2,21 @@
 // compile-flags:--test
 // normalize-stdout-test: "src/test/rustdoc-ui" -> "$$DIR"
 
+// Crates like core have doctests gated on `cfg(not(test))` so we need to make
+// sure `cfg(test)` is not active when running `rustdoc --test`.
+
 /// this doctest will be ignored:
```

Comment:
Since you're inverting the `cfg` for both these doctests, their actual doc comments are now incorrect.

Reply:
These comments still look correct to me.

Reply:
Ah, you're right, this is still correct, my bad.